src/OrchardCore.Modules/OrchardCore.Facebook/StartupLogin.cs

```diff
@@ -0,0 +1,37 @@
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Facebook;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Options;
+using OrchardCore.DisplayManagement.Handlers;
+using OrchardCore.Facebook.Login.Configuration;
+using OrchardCore.Facebook.Login.Drivers;
+using OrchardCore.Facebook.Login.Services;
+using OrchardCore.Modules;
+using OrchardCore.Navigation;
+using OrchardCore.Settings;
+
+namespace OrchardCore.Facebook
+{
+    [Feature(FacebookConstants.Features.Login)]
```

Comment:
If it's a new feature, it should have an entry in the manifest

Reply:
The new feature is the  [Feature(FacebookConstants.Features.Widgets)] and has an entry in the manifest :)

![image](https://user-images.githubusercontent.com/9623255/58986617-78c64b80-87e6-11e9-87c0-37a173bff819.png)

The core feature contains the settings needed to integrate the FB application and register fb scripts with resource manager
![image](https://user-images.githubusercontent.com/9623255/58986894-05710980-87e7-11e9-8c3d-f4f68e6ad1e2.png)

The login feature registers the FB Login external provider, using the appid and secret defined in the Application (core module feature)

The Widgets feature registers most of the fb plugins as widgets and a FacebookPlugin part to extend with new/missing fb plugins as widgets